src/routes/RequestsRoute.test.js

```diff
@@ -24,7 +24,7 @@ import {
 } from '../utils';
 import {
   REQUEST_LEVEL_TYPES,
-  createModes,
+  createModes, DEFAULT_REQUEST_TYPE_VALUE,
```

Comment:
Could you please move `DEFAULT_REQUEST_TYPE_VALUE,` to new line?

Reply:
Done